uktrade/data-hub-frontend
# Remove disabled administrative areas from State/Province selectors

## Description of change

Changes the Administrative Area drop-down (e.g. US State/ Canadian Province) to not show administrative areas that have been disabled.

## Test instructions

Update your API to have a disabled administrative area.
Attempt to Edit/Add a company with the relevant country for your administrative area.
You should not have the disabled area available as an option.

## Screenshots
### Before

(Alabama has been marked as disabled)
![image](https://user-images.githubusercontent.com/20663545/123652766-7ea38500-d824-11eb-9451-414739ea0923.png)


### After

(Alabama has been marked as disabled)
![image](https://user-images.githubusercontent.com/20663545/123652555-4e5be680-d824-11eb-9666-faec4e7d54a7.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
